backend/backend.sln

```diff
@@ -64,6 +66,10 @@ Global
 		{25F1F093-B645-4E30-95B3-7C82E5D2C380}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{25F1F093-B645-4E30-95B3-7C82E5D2C380}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{25F1F093-B645-4E30-95B3-7C82E5D2C380}.Release|Any CPU.Build.0 = Release|Any CPU
+		{A79EA7B2-1424-4AD1-9402-ADE965607962}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
are these added automatically? do you know what they specify?

Reply:
automatic, never touched an sln file manually (only via `dotnet sln add`)